mingw-w64-git/git-wrapper.c

```diff
@@ -250,7 +297,7 @@ static LPWSTR fixup_commandline(LPWSTR exepath, LPWSTR *exep, int *wait,
 		}
 
 		if (*p == L' ' && p[wcsspn(p, L" ")] == L'"' && space - cmd >= 8 &&
-		    !wcsncmp(space - 8, L"\\cmd.exe /K \"", 13)) {
+			!wcsncmp(space - 8, L"\\cmd.exe /K \"", 13)) {
```

Comment:
Please drop this white-space-only change from this PR. (The formatting should resemble Git's coding style, which wants the `!` to be aligned just after the opening parenthesis in the preceding line, read: the suggested change is actually incorrect.)